tests/files/tests.py

```diff
@@ -340,6 +341,37 @@ def test_file_move_overwrite(self):
         os.close(handle_a)
         os.close(handle_b)
 
+    def test_file_move_copystat_cifs(self):
+        handle_a, self.file_a = tempfile.mkstemp()
+        handle_b, self.file_b = tempfile.mkstemp()
+
```

Comment:
Making this change would make this test inconsistent with the formatting of the sibling `test_file_move_overwrite` test. This other test has been formatted in this same manner since at least Django 1.6.0 (I stopped checked history at this point). Given this history, we emulated the format of this older test in these newly added tests for consistency sake.

One option is to expand the scope of the PR to align all three tests for `file_move_safe` to same formatting. It seems like this would fall outside of the scope of this original PR to fix a blocking bug. I'm hesitant to expand the scope of this PR to include changes not directly related to fixing this blocking bug without a larger consensus from other reviewers.

What are people's feelings about changing the formatting of other tests in this suite that are not directly related to what this PR is trying to address (a blocker bug fix)?